src-java/kilda-persistence-tinkerpop/src/main/java/org/openkilda/persistence/ferma/frames/FlowStatsFrame.java

```diff
@@ -0,0 +1,90 @@
+/* Copyright 2020 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.persistence.ferma.frames;
+
+import static java.lang.String.format;
+
+import org.openkilda.model.Flow;
+import org.openkilda.model.FlowStats.FlowStatsData;
+
+import com.syncleus.ferma.VertexFrame;
+import com.syncleus.ferma.annotations.Property;
+import org.apache.tinkerpop.gremlin.structure.Direction;
+import org.apache.tinkerpop.gremlin.structure.Edge;
+
+import java.util.List;
+import java.util.Objects;
+
+public abstract class FlowStatsFrame extends KildaBaseVertexFrame implements FlowStatsData {
+    public static final String FRAME_LABEL = "flow_stats";
+    public static final String HAS_BY_EDGE = "has";
+    public static final String FLOW_ID_PROPERTY = "flow_id";
```

Comment:
Can we get rid of `flow_id` field into this table/class? Its existence forces us to track changes and provide updates in all copies by "hands".